### PR TITLE
Tasks: reorder tasks in main page

### PR DESCRIPTION
**Pull request by merveenoyan**
Move multimodal and nlp up 🔝 @julien-c 

thought it would be enough to change the order here seeing this [LoC](https://github.com/huggingface-internal/moon-landing/blob/9ab039334aed765cf07c0cad002c08ccb670b3e4/server/views/pages/TasksPage/TasksPage.svelte#L36) LMK in case I'm missing something

### CODE REVIEW

**Comment by merveenoyan** [comment]
closes https://github.com/huggingface-internal/moon-landing/issues/11505

**Review by Vaibhavs10 — APPROVED** [review]
Niceee! Atleast based on the structure of hf.co/tasks it looks the same and I wasn't able to find any other reference to it in the codebase (from a quick search).

**Comment by merveenoyan** [comment]
waiting for an approval from code owners

**Review by Wauplin — APPROVED** [review]

**Pull request merged**
